src/dom.ts

```diff
@@ -11,7 +11,7 @@ import {
 	restoreWorkingKey,
 } from './working-key';
 import observeTag from './observe-tag';
-import processEffects from './process-effects';
+import processHooks from './process-hooks';
```

Comment:
renamed this file / function, which better reflects what it does now (since it processes both effects and state!)